recipes/open62541/all/conanfile.py

```diff
@@ -329,6 +347,8 @@ def generate(self):
         # Honor BUILD_SHARED_LIBS from conan_toolchain (see https://github.com/conan-io/conan/issues/11840)
         tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0077"] = "NEW"
 
+        if version >= "1.4.11.1":
+            tc.cache_variables["UA_ENABLE_NODESETLOADER"] = self.options.get_safe("nodeset_loader", False)
```

Comment:
Lines above, it's removing the option for every OS but Linux:

```python
        # NodesetLoader has only rudimentary Windows support --> disabling for now. This might change in the future.
        if Version(self.version) < "1.4.11.1" or self.settings.os != "Linux":
            del self.options.nodeset_loader
```
So it explains why the CI macOS machines were failing (cc @AbrilRBS) and the purpose of this line is to avoid getting `None` instead of a boolean one:

```python
tc.cache_variables["UA_ENABLE_NODESETLOADER"] = self.options.get_safe("nodeset_loader", False)
```
Now, my question is, should we change the lines above or these into

```suggestion
        if version >= "1.4.11.1" and self.settings.os == "Linux":
            tc.cache_variables["UA_ENABLE_NODESETLOADER"] = self.options.get_safe("nodeset_loader")
```
?